Cargo.toml

```diff
@@ -30,13 +30,13 @@ dlc = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "69d63e1" }
 p2pd-oracle-client = { git = "https://github.com/p2pderivatives/rust-dlc", rev = "69d63e1" }
```

Comment:
I think we also need to update our rust-dlc lightning dependencies.

Reply:
It's not needed, I've tested it locally.